modules/table/src/test/java/org/apache/ignite/internal/table/distributed/TableManagerTest.java

```diff
@@ -448,9 +446,9 @@ public void testDropTable() throws Exception {
         assertNull(tableManager.table(DYNAMIC_TABLE_FOR_DROP_NAME));
         assertEquals(0, tableManager.tables().size());
 
-        verify(mvTableStorage, atMost(0)).destroy();
-        verify(txStateStorage, atMost(0)).destroy();
-        verify(replicaMgr, atMost(0)).stopReplica(any());
+        verify(mvTableStorage, never()).destroy();
+        verify(txStateStorage, never()).destroy();
+        verify(replicaMgr, never()).stopReplica(any());
```

Comment:
On line 456, `txStateStorage.destroy()` should not be called, but it is, according to the verification

Reply:
Why shouldn't it be called?

Reply:
Let me be more precise: it should be called only for non-colocation case. In case of colocation, tx state storage belongs to the zone, not the table, and this test does not destroy the zone.